11/jdk/slim-aws/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM openjdk:11.0.1-jdk-slim
+MAINTAINER Forest Keepers
+
+# Default to UTF-8 file.encoding
+ENV LANG C.UTF-8
+
+ENV AWSCLI_VERSION 1.16.47
+
+RUN apt-get update && \
+    apt-get install -y \
+      bzip2 \
+      unzip \
+      git \
+      wget \
+      curl \
+      python \
+      python-pip \
+      bash && \
+    pip install --no-cache-dir awscli==$AWSCLI_VERSION && \
+#    apt-get --purge autoremove python-pip -y && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get clean 
+
+RUN echo 'source $HOME/aws/env/bin/activate' >> .bashrc && \
```

Comment:
I think we do not need this, containers are not for logging in and play around :)

Reply:
Removed them